tools/bootstrap/node_.ps1

```diff
@@ -0,0 +1,56 @@
+# bootstrap/node_.ps1
+#
+# Node bootstrapping script for Windows.
+#
+# Automatically downloads a Node version to a cache directory and invokes it.
+#
+# The underscore in the name is so that typing `bootstrap/node` into
+# PowerShell finds the `.bat` file first, which ensures this script executes
+# regardless of ExecutionPolicy.
+$ErrorActionPreference = "Stop"
+
+function ExtractVersion {
+	param([string] $Path, [string] $Key)
+	foreach ($Line in Get-Content $Path) {
+		if ($Line.StartsWith("export $Key=")) {
+			return $Line.Substring("export $Key=".Length)
+		}
+	}
+}
+
+# Convenience variables
+$Bootstrap = Split-Path $script:MyInvocation.MyCommand.Path
```

Comment:
Major hiccup here
![image](https://user-images.githubusercontent.com/8171642/101848422-70f60e80-3b23-11eb-9dfc-80d9a78cb251.png)

We shouldn't be caching anything in the game directory (the guid) as it is thrown out every compile. The script CWD when `Build.bat` is called is fair game for caching though.

Reply:
Also i do have `node` in my PATH, why is it not using that?

Reply:
Alright, PreCompile.{sh,bat} now set an environment variable which overrides the cache directory used by node{,_.ps1}